src/utils/config/risk-framework.json

```diff
@@ -0,0 +1,711 @@
+{
```

Comment:
Just out of curiosity, how are these scores generated? Is it manual for each type of strategy?